luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -158,7 +160,7 @@ def sv_type(self):
         sorted_transcript_consequences, sv_type,
     ])
     def transcript_consequence_terms(self):
-        return self.mt.sortedTranscriptConsequences.map(lambda x: x[MAJOR_CONSEQUENCE]).extend([self.mt.svType])
+        return hl.set(self.mt.sortedTranscriptConsequences.map(lambda x: x[MAJOR_CONSEQUENCE]).extend([self.mt.svType]))
```

Comment:
This actually deviates from the previous behavior: https://github.com/broadinstitute/seqr-loading-pipelines/blob/4d0eb3486e14da9664a1c00810e395e0dbf6bfd3/sv_pipeline/genome/load_data_tests.py#L162

I think it's reasonable to not have duplicates though!

Reply:
Also, I checked the old index and it also has duplicates!

Reply:
okay well I guess we didn't have to change this then, but yay making things better!